Various/cool_MK ReSampler.lua

```diff
@@ -1,9 +1,26 @@
 -- @description MK ReSampler
 -- @author cool
--- @version 0.9
--- @changelog Initial
+-- @version 0.91
```

Comment:
Are you sure about 0.91? That's a lot higher than .9. Anything below 91 won't be allowed in future updates. Version numbers are not decimals.

Did you mean 0.9.1?

Reply:
0.9.1 - looks OK. I will re-upload this script with the correct version number. Thank you.

Version numbers are hell for me. I always thought that these were just tenths and hundredths of a unit, as in normal counting in small steps. I assumed that 0.91 is greater than 0.90 by only 0.01.

Reply:
Version numbers are an infinite list of number of text segments separated by any non-alphanumeric character (eg. 2024-01-22 is a fully valid version number). Each segments are compared individually and text segment (pre-release indicator) sort lower than numbers.